torch/csrc/jit/serialization/import.cpp

```diff
@@ -291,6 +330,23 @@ Module import_ir_module(
   return deserializer.deserialize(device, extra_files);
 }
 
+// For reading unified serialization format from torch.Package
+Module import_ir_module(
+    std::shared_ptr<CompilationUnit> cu,
+    PyTorchStreamReader& reader_,
+    c10::optional<at::Device> device,
+    std::string ts_id) {
+  std::string ts_package_location = ".data/ts_code/" + ts_id + "/";
+  ScriptModuleDeserializer deserializer(
+      std::move(cu),
+      reader_,
+      ts_package_location + "code/",
```

Comment:
This suggests that the code prefix will be ScriptModule-specific, but we want to share code between all pickled ScriptModules, right?